src/Microsoft.SqlTools.ServiceLayer/Localization/sr.xlf

```diff
@@ -2250,6 +2250,21 @@
         <target state="new">scripting</target>
         <note></note>
       </trans-unit>
+      <trans-unit id="BackupPathIsFolderError">
+        <source>The provided path specifies a directory but a file path is required: {0}</source>
+        <target state="new">The file name specified is also a directory name: {0}</target>

```

Comment:
I changed the text of this string but I only see that change in "source" and "target" still has the old string. does anyone know why?